common/services/court-hearing.js

```diff
@@ -13,19 +13,24 @@ const courtHearingService = {
 
       return value
     })
-  },
+  }
 
-  create(data, disableSaveToNomis = false) {
+  const create = (data, disableSaveToNomis = false) => {
     const params = {}
 
     if (disableSaveToNomis) {
       params.do_not_save_to_nomis = true
     }
 
     return apiClient
-      .create('court_hearing', courtHearingService.format(data), params)
+      .create('court_hearing', format(data), params)
```

Comment:
Leaving aside the fact that we don't appear to check that we call `format` on the data before passing it to the api client here (for other services we do have such tests), not using the form `courtHearingService.format` (which I presume is because we're now returning an instance) is going to make this hard (if not impossible) to test since the `format` is within the closure.

Maybe this should now be a good old class that you simply pass the req object to? And then we can use `this.format` instead.